src/WinRT.Runtime/Interop/IUnknownVftbl.cs

```diff
@@ -4,6 +4,14 @@
 using System;
 using System.Runtime.InteropServices;
 
+namespace WinRT
+{
+    internal struct VftblPtr

```

Comment:
Do you still need this?

Reply:
You're right, we don't. Removed it entirely 😄